frontend/simple/views/containers/Proposals.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <dashboard-section title="Active Proposals">
+
+    <!--  Start Original Voting Banner to delete soon -->
+    <voting-banner v-for="proposal in Object.values(proposals)"
+      :proposal="proposal"
+    />
+    <!--  End Original Voting Banner to delete soon -->
+
+    <h4 class="title is-size-5 notification is-warning gi-is-banner gi-notify"
+      v-if="groupProposals.notVoted.length"
+    >
+      <i18n>These are waiting for your vote!</i18n>
+    </h4>
+    <voting
+      v-for="proposal in groupProposals.notVoted"
+      :proposal="proposal"
+    />
+
+
+    <h4 class="title is-size-5" v-if="groupProposals.own.length">
+      <i18n>Your own proposals</i18n>
+    </h4>
+    <voting
+      v-for="proposal in groupProposals.own"
+      :proposal="proposal"
+    />
+
+    <i18n class="notification gi-is-banner gi-notify"
+      v-if="!showOtherProposals && allVoted"
+    >
+      Cool, you already voted on all active proposals.
+    </i18n>
+
+    <a class="gi-showOther has-text-weight-bold"
+      v-if="!showOtherProposals && groupProposals.alreadyVoted.length"
+      @click.prevent="showOtherProposals = true"
+    >
+      <i18n>Show</i18n> {{groupProposals.alreadyVoted.length}} <i18n>voted proposals</i18n>
+    </a>
+
+    <h4 class="title is-size-5" v-if="showOtherProposals">
+      <i18n>Voted proposals</i18n>
+    </h4>
+    <voting
+      v-if="showOtherProposals"
+      v-for="proposal in groupProposals.alreadyVoted"
+      :proposal="proposal"
+    />
+  </dashboard-section>
+</template>
+<style lang="scss" scoped>
+@import "../../assets/sass/theme/index";
+
+.gi-notify {
+  display: block;
+  margin-top: $gi-spacer-lg;
+}
+
+</style>
+<script>
+import DashboardSection from '../components/DashboardSection.vue'
+import Voting from '../components/Voting.vue'
+import VotingBanner from './VotingBanner.vue'
+// import ShowProposals from './ShowProposals.vue'
+
+export default {
+  name: 'Proposals',
+  components: {
+    DashboardSection,
+    Voting,
+    VotingBanner
+    // ShowProposals
+  },
+  props: {
+    proposals: Object
```

Comment:
I'm not sure whether `proposals` here, as a property, will result in a reactive binding to the `currentGroupState.proposals` vuex state that's being passed in.

The thing is, it needs to be a reactive binding (not a copy). So, first, check whether that's the case.

Then, consider simply directly referencing the `$store` from this component instead of passing in `currentGroupState.proposals` as a property, since that references the `$store` anyway.

Reply:
👍 that will be my part, and I plan to remove that prop and reach the store directly from this container, exactly as you said :) right now this container works with mocked data bc this pr focuses primarily on the display component, so I think it's fine to keep this as is for now

Reply:
Exactly what aniko said 😊